chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -150,7 +150,8 @@ def self.unclaimed(node, include_mounted=false)
               %x{lsblk #{d.name.gsub(/!/, "/")} --noheadings --output MOUNTPOINT | grep -q -v ^$}
               next if $?.exitstatus == 0
             end
-            d.fixed and not d.claimed?
+            next unless d.fixed || d.multipath?
```

Comment:
The logic of these two lines looks correct, but it was quite hard for me to follow.  Please could you add a comment or two?